### PR TITLE
creditBureau-Integration

### PR DESCRIPTION
**Pull request by rrpawar96**
## Description:
This PR has functionalities of : 
https://github.com/openMF/community-app/pull/3236
https://github.com/openMF/community-app/pull/3302

## Related issues and discussion
# https://github.com/openMF/web-app/issues/1270

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

**Comment by rrpawar96** [comment]
@karantakalkar, @BLasan Please Review.

**Comment by rrpawar96** [comment]
Documentation/Screenshots : https://cwiki.apache.org/confluence/pages/worddav/preview.action?fileName=Web-App+Documentation+to+use+the+Integrated+Credit+Bureau.pdf&pageId=173081071

**Comment by rrpawar96** [comment]
@Abhirup-99, Could you please help in reviewing and merge this code?

**Comment by Abhirup-99** [comment]
Hi, I can review but I don't have merge access.

**Comment by rrpawar96** [comment]
ya after this PR gets 'LGTM' from you, then will tell someone else to merge this code, Thanks. 👍

**Comment by bharathcgowda** [comment]
@luckyman20  can you please help in the review of this PR as well, this is the same Credit bureau PR that you reviewed on community-app.

File: src/app/clients/clients-routing.module.ts [hunk 1]
```diff
@@ -48,6 +51,9 @@ import { ClientActionsResolver } from './common-resolvers/client-actions.resolve
 import { ClientChargeViewResolver } from './common-resolvers/client-charge-view.resolver';
 import { ClientTransactionPayResolver } from './common-resolvers/client-transaction-pay.resolver';
 import { ClientDataAndTemplateResolver } from './common-resolvers/client-and-template.resolver';
+import { CreditBureauResolver } from './common-resolvers/credit-bureau.resolver';
+import { CreditBureauConfigurationResolver } from './common-resolvers/creditBureau-configuration.resolver';
+import {UploadCreditReportComponent} from './clients-view/credit-report/upload-CreditReport/upload-CreditReport.component';
```
Comment (luckyman20) [inline src/app/clients/clients-routing.module.ts:56]:
```suggestion
import { UploadCreditReportComponent } from './clients-view/credit-report/upload-CreditReport/upload-CreditReport.component';
```

File: src/app/clients/clients-routing.module.ts [hunk 2]
```diff
@@ -255,7 +269,37 @@ const routes: Routes = [
...
+  },
+    {
+      path: 'creditReport',
+      data: { title: extract('Clients'), breadcrumb: 'Clients', routeParamBreadcrumb: false },
+      children: [
+        {
+          path: 'THITSAWORKS',
+          component: ThitsaworksComponent,
+          data: { title: extract('THITSAWORKS'), breadcrumb: 'THITSAWORKS', routeParamBreadcrumb: false },
+         /* resolve: {
```
Comment (luckyman20) [inline src/app/clients/clients-routing.module.ts:281]: Are you sure we want these commented out code in future?

File: src/app/clients/clients-view/credit-report/credit-report.component.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
+<div class="tab-container mat-typography">
+  <div class="container">
+
+    <div class="m-b-10">
+      <h3>Credit Bureau</h3>
+    </div>
+    <!--
```
Comment (luckyman20) [inline src/app/clients/clients-view/credit-report/credit-report.component.html:7]: Please address this comment.

File: src/app/clients/clients-view/credit-report/credit-report.component.ts [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+  upload() {
+    this.creditBureauName = this.CreditBureauForm.value.creditBureauName;
+    this.router.navigate(['../creditReport/upload/']);
+  }
+
+
+
+}
+
+
```
Comment (luckyman20) [inline src/app/clients/clients-view/credit-report/credit-report.component.ts:77]: There shouldn't be any empty lines at the End of the file.

**Review by luckyman20 — COMMENTED** [review]
Please fix these formatting errors and address the comments. Note that I haven't gone through the whole PR but you can once check formatting errors and comments and address them, if possible.

**Pull request closed without merging**